isic/ingest/admin.py

```diff
@@ -257,8 +258,6 @@ def human_blob_size(self, obj):
     @admin.action(description='Extract zip')
     @takes_instance_or_queryset
     def extract_zip(self, request, queryset):
-        from isic.ingest.tasks import extract_zip as extract_zip_task
```

Comment:
I'm not sure why this needed to be scoped to the function. It doesn't seem to be a circular import.